boto3/data/ec2/2016-11-15/resources-1.json

```diff
@@ -2577,6 +2618,50 @@
           }
         }
       }
+    },
+    "VpnGateway": {
+      "identifiers": [
+        {
+          "name": "VpnGatewayId"
```

Comment:
I left similar feedback in another pull request. The identifier should be `Id` instead of `VpnGatewayId`; avoid stuttering the resource name into the identifier. Also, there should be a `memberName` attribute on the identifier of `VpnGatewayId`. This will require a few related changes in the resource model to update uses of VpnGatewayId.